src/drivers/cmakeDriver.ts

```diff
@@ -1566,11 +1566,13 @@ export abstract class CMakeDriver implements vscode.Disposable {
         const timeStart: number = new Date().getTime();
         const child = await this._doCMakeBuild(targets, consumer, isBuildCommand);
         const timeEnd: number = new Date().getTime();
+        const duration: number = timeEnd - timeStart;
```

Comment:
The only problem is that in the case of "cmake.buildTask": true" the user may need to select a task (if there is no default task) in `_doCMakeBuild` that will be calculated as the build time.